vaccine/data/medscheme.py

```diff
@@ -15,21 +15,14 @@ def data(self):
 
     @cached_property
     def schemes(self):
-        schemes = [(i["value"], i["text"]) for i in self.data]
-        schemes.sort(key=itemgetter(1))
-        lowercase = {v.strip().lower(): k for k, v in schemes}
-        schemes_kv = {k: v for k, v in schemes}
-        return lowercase, schemes_kv
+        return {i["value"]: i["text"] for i in self.data}
 
     def search_for_scheme(self, search_text):
-        search_text = search_text.strip().lower()
-        lowercase, _ = self.schemes
-        possibilities = difflib.get_close_matches(search_text, lowercase.keys(), n=3)
-        return [(lowercase[p], self.scheme_name(lowercase[p])) for p in possibilities]
+        possibilities = process.extract(search_text, self.data, limit=3)
```

Comment:
Interesting that we can use `self.data` here (a list of dictionaries), I would've expected us to have to use `self.schemes` (a dictionary of IDs and values to search)

Reply:
Very true - should I change this?